eureka2-write-server/src/main/java/com/netflix/eureka2/server/registry/EvictionQuotaProvider.java

```diff
@@ -1,10 +1,10 @@
-package com.netflix.eureka2.registry.eviction;
+package com.netflix.eureka2.server.registry;
 
 import rx.Observable;
 
 /**
  * Back pressure aware observable that emits eviction quotas, when requested
- * by {@link com.netflix.eureka2.registry.PreservableRegistryProcessor}.
+ * by {@link PreservableRegistryProcessor}.
  *
  * @author Tomasz Bak
  */
```

Comment:
Can we use a different name than Provider here? It conflates with Guice Provider<>.